### PR TITLE
x11 backend: set window title

### PR DESCRIPTION
**Pull request by agx**
This makes windows identifiable in the window list

### CODE REVIEW

File: include/backend/x11.h [hunk 1]
```diff
@@ -46,6 +46,8 @@ struct wlr_x11_backend {
 	struct {
 		struct wlr_x11_atom wm_protocols;
 		struct wlr_x11_atom wm_delete_window;
+	        struct wlr_x11_atom net_wm_name;
```
Comment (SirCmpwn) [inline include/backend/x11.h]: Weird indent here

File: backend/x11/backend.c [hunk 2]
```diff
@@ -221,11 +221,20 @@ static bool wlr_x11_backend_start(struct wlr_backend *backend) {
...
 	init_atom(x11, &x11->atoms.wm_delete_window, 0, "WM_DELETE_WINDOW");
+	init_atom(x11, &x11->atoms.net_wm_name, 1, "_NET_WM_NAME");
+	init_atom(x11, &x11->atoms.utf8_string, 0, "UTF8_STRING");
 
 	xcb_change_property(x11->xcb_conn, XCB_PROP_MODE_REPLACE, output->win,
 		x11->atoms.wm_protocols.reply->atom, XCB_ATOM_ATOM, 32, 1,
 		&x11->atoms.wm_delete_window.reply->atom);
 
+	char title[32];
+	if (snprintf(title, 32, "wlroots - %s", output->wlr_output.name))
```
Comment (SirCmpwn) [inline backend/x11/backend.c]: Add `{}` to this if statement please

Comment (emersion) [reply backend/x11/backend.c]: Also, can you use `sizeof(title)` so that there's no bad surprise if we change it?

Comment (agx) [reply backend/x11/backend.c]: All updated. Thanks for the review.

**Comment by SirCmpwn** [comment]
Thanks!

**Pull request merged**